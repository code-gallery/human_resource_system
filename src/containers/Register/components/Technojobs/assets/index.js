import technojobs from './technojobs-logo.jpg'

export default {
  technojobs
}
