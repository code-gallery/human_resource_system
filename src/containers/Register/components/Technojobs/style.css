.Technojobs {
  color: #fff;
}

.Technojobs img {
  height: 55px;
  display: inline-block;
  border-radius: 100px;
}
