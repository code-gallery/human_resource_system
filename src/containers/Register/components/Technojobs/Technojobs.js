import React from 'react'
import Assets from './assets'
import './style.css'

const Technojobs = () => {
  return (
    <div className="Technojobs form-group text-center">
      <p>In partnership with</p>
      <img src={Assets.technojobs} alt="Technojobs logo" />
    </div>
  )
}

export default Technojobs
