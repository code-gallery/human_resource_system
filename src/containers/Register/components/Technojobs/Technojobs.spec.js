import React from 'react'
import renderer from 'react-test-renderer'
import Technojobs from './index'

describe('<Technojobs />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Technojobs />
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
