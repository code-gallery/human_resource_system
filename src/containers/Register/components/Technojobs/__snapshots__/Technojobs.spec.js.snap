// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Technojobs /> renders correctly 1`] = `
<div
  className="Technojobs form-group text-center"
>
  <p>
    In partnership with
  </p>
  <img
    alt="Technojobs logo"
    src="technojobs-logo.jpg"
  />
</div>
`;
